README.md

```diff
@@ -81,23 +81,44 @@ Or install them with the `all` directive. `all` defaults to Spark 2.4.x currentl
 pip install "flytekit[all]"
 ```
 
-## Testing
+## Development
 
-Flytekit is Python 2.7+ compatible, so when feasible, it is recommended to test with both Python 2 and 3.
+### Recipes
+
+```
+$ make
+Available recipes:
+  setup        Install requirements
+  fmt          Format code with black and isort
+  lint         Run linters
+  test         Run tests
+  requirements Compile requirements
+```
 
-### Unit Testing
+### Setup (Do Once)
 
-#### Setup (Do Once)
 ```bash
 virtualenv ~/.virtualenvs/flytekit
 source ~/.virtualenvs/flytekit/bin/activate
-python -m pip install -r requirements.txt
-python -m pip install -U ".[all]"
+make setup
+```
+
+### Formatting
+
+We use [black](https://github.com/psf/black) and [isort](https://github.com/timothycrosley/isort) to autoformat code. Run the following command to execute the formatters:
+
+```bash
+source ~/.virtualenvs/flytekit/bin/activate
+make fmt
 ```
 
-#### Execute
+### Testing
+
+Flytekit is Python 2.7+ compatible, so when feasible, it is recommended to test with both Python 2 and 3.
```

Comment:
Let's kill this line... I've been actively removing Python 2.7 stuff.  I think the new version we're targeting will only be Python 3.6+.